tpot/tpot.py

```diff
@@ -23,13 +23,16 @@
 from .base import TPOTBase
 from .config_classifier import classifier_config_dict
 from .config_regressor import regressor_config_dict
+from .config_classifier_lite import classifier_config_dict_lite
+from .config_regressor_lite import regressor_config_dict_lite
 
 
 class TPOTClassifier(TPOTBase):
     """TPOT estimator for classification problems"""
 
     scoring_function = 'accuracy'  # Classification scoring
     default_config_dict = classifier_config_dict # Classification dictionary
+    lite_config_dict = classifier_config_dict_lite # Classification dictionary for TPOT lite
```

Comment:
Is storing the light config dictionary necessary here? Can't it just import and use it only when the user specifies to use TPOT light?